examples/sycl/00_pvc_gemm/CMakeLists.txt

```diff
@@ -34,3 +34,10 @@ cutlass_example_add_executable(
   TEST_COMMAND_OPTIONS
   TEST_BATCHES
 )
+
```

Comment:
Could you add something like:
`set(TEST_ODD_SHAPE --k=77)`
and pass that in place of (or in addition to) `TEST_BATCHES` for 00_pvc_gemm_padded?

Reply:
Oh and what about:
`set(TEST_SMALL_SHAPE --m=8 --k=8)` and append that for 00_pvc_gemm? Since we've removed the requirement that the tilesize should evenly divide the shape (I confirmed that works on BMG :+1: )

Reply:
Forget the `TEST_ODD_SHAPE` suggestion - as discussed offline it won't work on PVC.

Reply:
done